pom.xml

```diff
@@ -2451,7 +2461,7 @@
                       <groupId>tech.picnic.error-prone-support</groupId>
                       <artifactId>error-prone-contrib</artifactId>
                       <version>${error-prone-support.version}</version>
-                    </path>b
+                    </path>
```

Comment:
Please remove this extra char

Reply:
I noticed this got introduced in the previous PR, so I dropped it in this commit.